src/Routes.js

```diff
@@ -20,6 +22,8 @@ export default () => (
       <Route path='/sign-up' component={SignUp}/>
       <Route path="/task/create" component={Create} />
       <Route path="/task/update" component={Update} />
+      <Route path='/task/list' component={OverView} />,
```

Comment:
Maybe we should call the component `OverView` `List` instead, it is easier to reason about if the route matches the component name.